sqlglot/parser.py

```diff
@@ -693,10 +694,13 @@ def raise_error(self, message, token=None):
         self.errors.append(error)
 
     def expression(self, exp_class, **kwargs):
+        comments = kwargs.pop("comments", None)
```

Comment:
just add comment as a first class kwarg default nine

Reply:
none

Reply:
I cannot get this to work. You meant to changing

``
class Expression(metaclass=_Expression):
    arg_types = {"this": True, "comment": False}
``

correct? 

But that means changing ``arg_types `` for every class that bring it's own ``arg_types``, which is impractical.

Unless I misunderstood what you're suggesting ...

Reply:
no just to this method add

def expression(self, comment=None, **kwargs